install/kubernetes/helm/istio/README.md

```diff
@@ -160,12 +160,12 @@ Helm charts expose configuration options which are currently in alpha.  The curr
 | `global.arch.s390x` | Specifies the scheduling policy for `s390x` architectures | 0 = never, 1 = least preferred, 2 = no preference, 3 = most preferred | `2` |
 | `global.arch.ppc64le` | Specifies the scheduling policy for `ppc64le` architectures | 0 = never, 1 = least preferred, 2 = no preference, 3 = most preferred | `2` |
 | `ingress.enabled` | Specifies whether Ingress should be installed | true/false | `true` |
-| `gateways.istio-ingressgateway.enabled` | Specifies whether Ingress gateway should be installed | true/false | `true` |
-| `gateways.istio-egressgateway.enabled` | Specifies whether Egress gateway should be installed | true/false | `true` |
+| `gateways.enabled` | Specifies whether gateway should be installed | true/false | `true` |
```

Comment:
Don't delete those. They are still available to toggle either ingress or egress.
E.g. one might only want to install the ingressgateway.

Reply:
@ymesika Are you sure ? I couldn't find those variables !

Reply:
My bad! Updated the README again please check.